lab-Marshall/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "test",
+  "version": "0.0.1",
+  "devDependencies": {
```

Comment:
You need to include gulp as a devDependency. It needs to be installed locally as well as globally in order to run.